### DISCUSSION TITLE
Promotion in the social network "VKontakte" (https://vk.com)

### DISCUSSION THREAD

**Discussion by DEHNCKA** [d2phap/ImageGlass#1238] [Q&A] [answered]
I didn't find "ImageGlass" group in "VKontakte" network. Only rare mentions. It would be nice to create a group there for a Russian-speaking audience, this will help promote the application and give users the opportunity to contact developers in a more familiar environment.

**Comment by d2phap** [comment, marked as answer]
Thanks @DEHNCKA 
ImageGlass currently has Facebook, Instagram, Twitter, Google group, Discord, GitHub, LinkedIn, Crowndin, Patreon,... but I'm the only one who manage all of them (in my free time). @fire-eggs also helps me on GitHub and Google group. Recently, I had had to disable the chat feature of IG Facebook because I could not reply to all of them. Having more social network is nice but my time is limited. 😅
